src/pudl/metadata/fields.py

```diff
@@ -1999,6 +2042,23 @@
         },
         "technology_description": {"constraints": {"enum": set(TECH_DESCRIPTIONS)}},
     },
+    "transmission_ferc1": {
+        "capex_land": {
+            "description": "Cost of Land and land rights for the transmission line."
+        },
+        "capex_other": {
+            "description": "Construction and other costs for the transmission line."
+        },
+        "capex_total": {"description": "Total costs for the transmission line."},
+        "opex_operations": {
+            "description": "Operating expenses for the transmission line."
+        },
+        "opex_maintenance": {
+            "description": "Maintenance expenses for the transmission line."
+        },
+        "opex_rents": {"description": "Rent expenses for the transmission line."},
+        "opex_total": {"description": "Overall expenses for the transmission line."},
```

Comment:
can you add types for these?

Reply:
I wasn't sure if these needed types because they are already given types in the `FIELD_METADATA` dict. Does this `FIELD_METADATA_BY_RESOURCE` fully override whatever is in `FIELD_METADATA` or does it supplement / replace whatever is in there?

Reply:
oh your very right my bad!